content/06.administrative.md

```diff
@@ -0,0 +1,9 @@
+## Data and Resource Availability
+
+## Funding
+
+## Acknowledgements
```

Comment:
```suggestion
## Acknowledgments
```
Because you use the preferred American English spelling in the custom dictionary.